package-lock.json

```diff
@@ -1,7896 +0,0 @@
-{
```

Comment:
this file shouldn't be deleted, you can re-generate it with `npm install`

Reply:
Sorry forgot that got uploaded